tests/foreman/sys/test_fam.py

```diff
@@ -123,10 +123,15 @@ def common_fam_setup(satellite):
 
 
 @pytest.fixture(scope='module')
-def setup_fam(module_target_sat, module_sca_manifest, install_import_ansible_role):
+def setup_fam(
+    module_target_sat, module_sca_manifest, install_import_ansible_role, module_capsule_configured
+):
     # Execute AAP WF for FAM setup
     Broker().execute(workflow='fam-test-setup', source_vm=module_target_sat.name)
 
+    # Update the settings to point to our Capsule
+    settings.set('fam.server.foreman_proxy', module_capsule_configured.hostname)
```

Comment:
PRT run 12685 passed with `settings.fam.server.foreman_proxy = module_capsule_configured.hostname`
PRT run 12687 is running with `settings.set(…)`.

Personally, I prefer the `settings.fam… = …` notation, but @Gauravtalreja1 noted that it's not always working as intended.

Reply:
Thanks @evgeni